src/main/resources/application.properties

```diff
@@ -69,3 +69,5 @@ com.ampnet.identityservice.veriff.private-key=
 com.ampnet.identityservice.veriff.api-key=
 
 com.ampnet.identityservice.test.enabled-test-kyc=true
+
+com.ampnet.identityservice.provider.blockchain-api=https://eth-goerli.alchemyapi.io/v2/Geqo-GtNKkGpfFbcCR2-67gPzI7wpTRd
```

Comment:
This is a secret value and shouldn't be provided in public repo. If it is needed for testing we can provide it file `application-secret.properties` and it to `.gitignore`.

Reply:
This is just an api offered by Alchemy, it is not secret since anyone can easily open a similar api.

Reply:
Use it like this: `com.ampnet.identityservice.provider.blockchain-api=$ALCHEMY_API`